sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -117,7 +117,7 @@ statement
         tableIdentifier partitionSpec?                                 #loadData
     | TRUNCATE TABLE tableIdentifier partitionSpec?
         (COLUMNS identifierList)?                                      #truncateTable
-    | ADD identifier .*?                                               #addResource
+    | op=(ADD | DELETE | LIST) identifier .*?                          #manageResource
```

Comment:
Yes. I was thinking that another PR will handle the implementation of it. So I add the unsupported exception in SparkSqlParser in this PR. But I will leave it to another PR for parser also. Thanks! Will update.